docs/testing-operators.md

```diff
@@ -104,36 +119,47 @@ export QUAY_TOKEN="basic abcdefghijkl=="
 The following command will run `operator-courier` against the bundle directory `my-operator/` from the example above.
 
 ```
-operator-courier verify --ui_validate_io my-operator/
+$ operator-courier verify --ui_validate_io <PATH-OLM-FILES> # E.g operator-courier verify --ui_validate_io my-operator/deploy/olm-catalog/my-operator/
```

Comment:
In the beginning of the document we explain that `my-operator/` is a reference to the bundle directory that we established in the very beginning. The path used here is from the SDK output, which might not apply to all folks who read this document and breaks with the convention declared in the first part of this doc.

Reply:
HI @dmesser,

Thank you for your review. I did some more few changes based on your comment. 

I fought very confuse this nomenclature my-operator to tell the user that we are expecting the dir where the files are created. Mainly because we are using the same to describe the operator dir in the following part. 

<img width="996" alt="Screenshot 2019-09-05 at 14 44 36" src="https://user-images.githubusercontent.com/7708031/64347528-008d4300-cfec-11e9-8113-5dcfed22f861.png">

Also, IMHO it would be easier to understand if we use the default dir path which is where the operator-SDK will create the files since in all docs we are suggestion using the tool to do this. 

Please, let me know WDYT over it now after this small changes? Are you ok with move forward? Have you any suggestion over it that you would like to address here in this PR?